internal/ignition/ignition.go

```diff
@@ -1332,6 +1332,11 @@ func (ib *ignitionBuilder) FormatSecondDayWorkerIgnitionFile(address string, mac
 	return buf.Bytes(), nil
 }
 
+func QuoteSshPublicKeys(sshPublicKeys string) string {
+	sshPublicKeysList := strings.Split(sshPublicKeys, "\n")
+	return strings.Join(sshPublicKeysList, `", "`)
```

Comment:
WDYT about using `strings.ReplaceAll` instead? That should save one slice scan.

Reply:
Good idea, thanks!